octoprint_mrbeam/software_update_information.py

```diff
@@ -1,69 +1,178 @@
-from datetime import datetime, date
+import json
 import os, sys
+import threading
+from datetime import date, datetime
+from os.path import join
 
-from octoprint.util import dict_merge
-from octoprint_mrbeam import IS_X86
+import requests
+
+from octoprint_mrbeam import MrBeamEvents
 from octoprint_mrbeam.mrb_logger import mrb_logger
-from octoprint_mrbeam.util import logExceptions
+from octoprint_mrbeam.util.log import logme
 from util.pip_util import get_version_of_pip_module
 
-
 SW_UPDATE_TIER_PROD = "PROD"
-SW_UPDATE_TIER_BETA = "BETA"
 SW_UPDATE_TIER_DEV = "DEV"
+SW_UPDATE_TIER_BETA = "BETA"
 DEFAULT_REPO_BRANCH_ID = {
     SW_UPDATE_TIER_PROD: "stable",
     SW_UPDATE_TIER_BETA: "beta",
     SW_UPDATE_TIER_DEV: "develop",
 }
 
+SW_UPDATE_FILE = "update_info.json"
+SW_UPDATE_CLOUD_PATH = "https://mr-beam.org/beamos/config/sw-update-conf"
+
 # add to the display name to modules that should be shown at the top of the list
 SORT_UP_PREFIX = " "
 
-
 _logger = mrb_logger("octoprint.plugins.mrbeam.software_update_information")
 
+sw_update_config = dict()
+
```

Comment:
Please do NOT use mutable globals as much as possible.

Reply:
I looked it up I used it as it was before your buster changes, I can change this like you did for the buster image